group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -7160,6 +7881,34 @@ private int shareGroupHeartbeatIntervalMs(String groupId) {
             .orElse(config.shareGroupHeartbeatIntervalMs());
     }
 
+    /**
+     * Get the session timeout of the provided streams group.
+     */
+    private int streamsGroupSessionTimeoutMs(String groupId) {
+        return 45000;
```

Comment:
These are left constant for now. To be replaced by calls to the `GroupConfigManager` when @aliehsaeedii introduces the group configurations.